static/js/tools/map/chart_loader.tsx

```diff
@@ -134,6 +141,46 @@ export function ChartLoader(): JSX.Element {
   );
 }
 
+function getGeoJsonDataFeatures(
+  placeDcids: string[],
+  enclosedPlaceType: string
+): GeoJsonFeature[] {
+  const distance = MANUAL_GEOJSON_DISTANCES[enclosedPlaceType];
+  if (!distance) {
+    return [];
+  }
+  const geoJsonFeatures = [];
+  for (const placeDcid of placeDcids) {
+    if (!placeDcid.includes("/") || !placeDcid.includes("_")) {
```

Comment:
I suspect there might be slowness from doing 2 string includes and 2 string splits over all the dcids.

Reply:
done: removed the 2 string includes